Process.h

```diff
@@ -155,8 +155,13 @@ typedef struct Process_ {
     * Linux:
     *   - from file capabilities
     *   - inherited from the ambient set
+    *
+    * Values:
+    *    0:  inital value
```

Comment:
There's a typo here. And it's better to state the meaning of the value in addition to how we can get the value.
```suggestion
    *    0:  initial value / undetermined
```

Reply:
thanks, adjusted